win32ss/gdi/gdi32/objects/text.c

```diff
@@ -465,9 +465,16 @@ ExtTextOutA(
     UNICODE_STRING StringU;
     BOOL ret;
 
-    RtlInitAnsiString(&StringA, (LPSTR)lpString);
+    if (fuOptions & ETO_GLYPH_INDEX)
+        return ExtTextOutW(hdc, x, y, fuOptions, lprc, (LPCWSTR)lpString, cch, lpDx);
+
+    StringA.Buffer = (PCHAR)lpString;
+    StringA.Length = StringA.MaximumLength = cch;
     RtlAnsiStringToUnicodeString(&StringU, &StringA, TRUE);
 
+    if (StringU.Length != (StringA.Length * 2))
+        DPRINT1("ERROR: Should convert lpDx properly!\n");
```

Comment:
Is this more like a "todo" for this ExtTextOutA function?

Reply:
yeah, this should receive some special handling, but was not hit when running some tests and this application so I left it.

Reply:
Can `StringA` be a multi-byte string?

Reply:
> Can `StringA` be a multi-byte string?

If so, then `RtlMultiByteToUnicodeN` and `RtlMultiByteToUnicodeSize` may be of interest.

Reply:
In general, a multi-byte character (of multiple bytes) can be converted to one `WCHAR`. Now, the `A` suffix of `ExtTextOutA` means multi-byte version instead of ANSI version.

Reply:
I have no interest in fixing this now, but it should probably be converted manually here, because the `lpDx` should be kept in-sync with it.